packages/frontend/src/pages/Token/BuyToken.tsx

```diff
@@ -0,0 +1,78 @@
+import { useQuoteToken } from 'hooks'
+import { BaseSyntheticEvent, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { PrimaryButton, SecondaryButton } from 'src/components/Button'
+import FloatInput from 'src/components/Input/FloatInput'
+import PercentInput from 'src/components/Input/PercentInput'
+import useMemecoin from 'src/hooks/useMemecoin'
+import Box from 'src/theme/components/Box'
+import { Column, Row } from 'src/theme/components/Flex'
+import * as Text from 'src/theme/components/Text'
+
+import * as styles from './style.css'
+
+const DEFAULT_SLIPPAGE = 0.5
+const DEFAULT_AMOUNT = 0
```

Comment:
These can be moved to the [`constants`](https://github.com/keep-starknet-strange/unruggable.meme/blob/main/packages/frontend/src/constants/misc.ts)